tests/test_infra_oncall.py

```diff
@@ -67,10 +67,14 @@ async def test_oncall_add(bot, plugin, db, respx_mock, ircnick):
     current_value = await db.fetch("SELECT * FROM oncall")
     assert len(current_value) == 1
     assert current_value[0]["username"] == "dummy"
-    if ircnick == "irc:///dummyirc":
-        assert current_value[0]["mxid"] == "@dummy:fedora.im"
-    else:
-        assert current_value[0]["mxid"] == "@dummymx:example.com"
+    assert (
```

Comment:
I recommend extending this test by adjusting the parametrize decorator. Something like:

```
@pytest.mark.parametrize(
    "response",
    [
        {"username": "username", "ircnicks": ["irc:///dummyirc"], "timezone": None},
        {"username": "dummy", "ircnicks": ["matrix://example.com/dummymx"]},
    ],
)
async def test_oncall_add(bot, plugin, db, respx_mock, ircnick):
    respx_mock.get("http://fasjson.example.com/v1/users/dummy/").mock(
        return_value=httpx.Response(
            200,
            json={"result": response},
        )
    )
```

This way you'll exercise both the timezone key being missing, and the timezone key being there, but `None`. The test should fail with the current change, and then when you add `or "UTC"` it should pass.

Reply:
Okayyy , got it!
I''l get to it now.